src/logging.c

```diff
@@ -150,6 +150,8 @@ static enum rcutils_get_env_retval rcutils_get_env_var_zero_or_one(
   const char * one_semantic)
 {
   const char * env_var_value = NULL;
+  (void)one_semantic;
+  (void)zero_semantic;
```

Comment:
These changes shall be done in the original repo: https://github.com/ros2/rcutils

Remember that this is a fork